src/asynk/async_queue.rs

```diff
@@ -330,20 +340,25 @@ where
     <Tls as MakeTlsConnect<Socket>>::TlsConnect: Send,
     <<Tls as MakeTlsConnect<Socket>>::TlsConnect as TlsConnect<Socket>>::Future: Send,
 {
-    pub async fn connect(
-        uri: impl ToString,
-        tls: Tls,
-        duplicated_tasks: bool,
-    ) -> Result<Self, AsyncQueueError> {
-        let manager = PostgresConnectionManager::new_from_stringlike(uri, tls)?;
-        let pool = Pool::builder().build(manager).await?;
-
-        Ok(Self {
-            pool,
-            duplicated_tasks,
-        })
+    pub fn check_if_connection(&self) -> Result<(), AsyncQueueError> {
```

Comment:
can be private I think?

Reply:
think is good to user , but if you want make it private ... as you wish.